packages/core/src/app/shipping/hasSelectedShippingOptions.ts

```diff
@@ -9,13 +9,7 @@ export default function hasSelectedShippingOptions(consignments: Consignment[]):
     return every(
         consignments,
         (consignment) =>
-            (consignment.selectedShippingOption &&
-                consignment.selectedShippingOption.id &&
-                // Selected option is available
-                consignment.availableShippingOptions &&
-                consignment.availableShippingOptions.filter(
-                    ({ id }) => id === consignment.selectedShippingOption?.id,
-                ).length) ||
+            (consignment.selectedShippingOption && consignment.selectedShippingOption.id) ||
```

Comment:
For context, this check is no longer unnecessary since the API automatically invalidates the selected shipping option if it's unavailable after updating the address.